compiler/luci-interpreter/src/loader/nodes/Conv2D.cpp

```diff
@@ -35,8 +35,9 @@ std::unique_ptr<Kernel> build_kernel_CircleConv2D(const luci::CircleNode *circle
   const Tensor *bias = helper.getOptionalInputTensor(node->bias());
   Tensor *output = helper.getOutputTensor(node);
 
-  auto scratchpad =
-    std::make_unique<Tensor>(input->element_type(), Shape({}), AffineQuantization{}, "");
+  // It is unknown what data will be stored in scratchpad tensor,
+  // using SINT8 as a most general option
+  auto scratchpad = std::make_unique<Tensor>(DataType::S8, Shape({}), AffineQuantization{}, "");
```

Comment:
`loco` provides special `DataType` value for case of unknown dtype.

```suggestion
  auto scratchpad = std::make_unique<Tensor>(DataType::Unknown, Shape({}), AffineQuantization{}, "");
```

Reply:
@m-bronnikov 
I still think it is better to stick to UINT8 and do not change it after tensor is created.

This scratch pad is used by PAL for different purposes and sometimes we can not even predict what is the actual data type is required and will be stored.
For example:
- [here](https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/NN/Source/ConvolutionFunctions/arm_convolve_s8.c#L136) it is used as 8 bit integer array
- [in the same function](https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/NN/Source/ConvolutionFunctions/arm_convolve_s8.c#L225) it is used as a 16 bit integer array

Reply:
I propose to present `scratchpad` to code outside of PAL is a simple blob of data, without any explicit semantics.

I think UINT8 data type and size in bytes looks the most appropriate option to represent scratchpad to "outer world".

Reply:
`UINT8` or `SINT8` https://github.com/Samsung/ONE/pull/8047#discussion_r758394641? =)

I like `loco::DataType::Unknown` because, as you mentioned above in comment line and in https://github.com/Samsung/ONE/pull/8069#discussion_r759210782, we don't really know which datatype will be used by implementation in PAL internally and `Unknown` looks more natural from reviewer point of view. 

However, I don't see any key points here and no matter for me what value will be setted by default. So, I agree with any approach.

Reply:
@m-bronnikov 
I prefer UINT8 (not a big different from SINT8, but internally it is used uint8, so I changes my mind from comment in 8047 :slightly_smiling_face: ) over Unknown, because Unknown semantically does not have "one element" size.
So we have kind of ambiguity if we use Unknown.

When we are using UINT8 we can see scratchpad as a simple buffer of bytes, which has it's size encoded in tensor shape.